pulp_rpm/app/modelresource.py

```diff
@@ -360,7 +360,7 @@ def set_up_queryset(self):
     class Meta:
         model = RpmRepository
         import_id_fields = ("name",)
-        fields = ("name", "pulp_type", "description", "original_checksum_types", "sub_repo")
+        fields = ("name", "pulp_type", "description", "original_checksum_types", "user_hidden")
```

Comment:
Double check me here because I'm not familiar with any of this ModelResource code.